cra-tf-validate-ignore-goals.json

```diff
@@ -0,0 +1,10 @@
+{
+    "scc_goals": [
+        {
+            "scc_goal_id": "3000220",
+            "description": "Check whether Databases for etcd network access is restricted to a specific IP range",
+            "ignore_reason": "This module supports restricting network access using Context Based Restrictions (CBRs), however SCC does not yet support scanning for CBR rules, hence the goal currently fails. SCC CBR support is being tracked in AHA SCC-961",
+            "is_valid": true
+        }
+    ]
+}
```

Comment:
We need to migrate to rules and not goals, and we need to migrate to CRA config in a file. See:

- https://github.ibm.com/GoldenEye/module-template/blob/master/cra-tf-validate-ignore-rules.json
- https://github.ibm.com/GoldenEye/module-template/blob/master/cra-config.yaml

Reply:
seems we already have the cra-config.yaml file, but its pointing to the wrong directory - needs to be fscloud example

Reply:
There is no fscloud example in the etcd module as of now. So what should I do?

Reply:
point it at the complete example until we add an fscloud example